### DISCUSSION TITLE
Missing Docker packages

### DISCUSSION THREAD

**Discussion by matmut7** [community/community#56237] [Packages]
### Select Topic Area

Bug

### Body

Our Github organization has recently encountered an issue with the Docker registry: some versions of our packages become unavailable but continue to be listed on the Github web interface.

For example, here is a package listed here: https://github.com/socialgouv/docker/pkgs/container/docker%2Fnginx4spa/87247106?tag=7.6.0

![image](https://github.com/community/community/assets/38255502/b85d617f-09b3-49eb-b4e6-39822d6d5c7a)

The package appears to be available on the website, yet if we try to pull it:

```
❯ docker pull ghcr.io/socialgouv/docker/nginx4spa:7.6.0
7.6.0: Pulling from socialgouv/docker/nginx4spa
manifest unknown
```

This has happened with several packages in our organization, but we have not found an obvious link between them.

**Comment by mvdstruijk** [comment]
Hello, have you managed to find a solution for this problem? I'm experiencing the same problem and would appreciate any insights you might have.

**Reply by matmut7** [reply]
No, we moved everything on our self-hosted Harbor 😅
It continued to happen from time to time

**Comment by patchwork01** [comment]
We've got the same problem. It seems to happen to all our images as soon as they're older than a certain age, which doesn't seem to be very old. For example:

https://github.com/gchq/sleeper/pkgs/container/sleeper-deployment/176582523?tag=0.21.0
https://github.com/gchq/sleeper/pkgs/container/sleeper-deployment/195478646?tag=0.22.1

The most recently built one still seems to work:

https://github.com/gchq/sleeper/pkgs/container/sleeper-deployment/versions

```
$ docker logout ghcr.io
Removing login credentials for ghcr.io
$ docker image pull ghcr.io/gchq/sleeper-deployment:0.22.1
0.22.1: Pulling from gchq/sleeper-deployment
manifest unknown
$ docker manifest inspect ghcr.io/gchq/sleeper-deployment:0.22.1
manifest unknown
$ docker manifest inspect ghcr.io/gchq/sleeper-deployment:0.21.0
manifest unknown
$ docker manifest inspect ghcr.io/gchq/sleeper-deployment:develop
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 3254,
         "digest": "sha256:adf980c91d2c5ae538f8332a8c94ff1669d2e3facdd7d18bb06d92c34ee8cf75",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 3254,
         "digest": "sha256:a3c6de52536cc71df289160feb9d81d70a3783bb3316a1692953d6ca8fc3e84f",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      }
   ]
}
```
